test/katex-spec.js

```diff
@@ -1755,8 +1755,8 @@ describe("An accent parser", function() {
         expect(parse.type).toEqual("supsub");
     });
 
-    it("should not parse expanding accents", function() {
-        expect("\\widehat{x}").toNotParse();
+    it("should parse expanding accents", function() {
+        expect("\\widehat{x}").toParse();
```

Comment:
I'm not sure how Jasmine would know whether or not a shift occurred. That sounds more like a Screenshotter thing. Unless I'm missing something?